contracts/axelar-gateway/src/error.rs

```diff
@@ -8,4 +8,14 @@ pub enum Error {
     EmptyMessages = 2,
     RotationAlreadyExecuted = 3,
     NotLatestSigners = 4,
+    InvalidOperators = 5,
```

Comment:
section out errors by relevant module for clarity (if there's a better error pattern, we'll switch to that in the future)

Reply:
got it, updated.